### DISCUSSION TITLE
UPDATE LOG_MINING_FLUSH SET LAST_SCN  fail

### DISCUSSION THREAD

**Discussion by believerHan** [apache/flink-cdc#2492] [Q&A]
In an Oracle 12c database, there exists a table named LOG_MINING_FLUSH. I can execute the UPDATE statement UPDATE LOG_MINING_FLUSH SET LAST_SCN = 13780583354171 using the DBeaver tool while connected with the same user in Oracle. However, when using Flink CDC Oracle 2.4.1 with Flink SQL, I get an error message 'ORA-00942: table or view does not exist

**Comment by joyce0323** [comment]
Is it solved? 19C has the same issue
